### PR TITLE
Upload John Hopkins/IHS test results

### PR DESCRIPTION
**Pull request by RickHawesUSDS**
This PR has the changes necessary to upload the John Hopkins/IHS data. 

The John Hopkins is conducting COVID tests in the Navajo nation. They have approached USDS for support for a one-time upload of these test results for AZ. AZ wants these tests in their secondary feed. 

## Changes
- John Hopkins sender and schema. Based on the AZ flat-file, but handling the differences. 
- A secondary feed for the AZ PHD
- A secondary feed file name format

### Testing
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [x] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Specific Security-related subjects a reviewer should pay specific attention to

- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
*List GitHub issues this PR fixes*
- #1663 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

**Review by RickHawesUSDS — COMMENTED** [review]
A ready has createDate in the sortable spot.

File: prime-router/metadata/file_name_templates/file-name-templates.yml [hunk 1]
```diff
@@ -8,6 +8,16 @@
...
+- name: standard_secondary
+  lowerCase: true
+  elements:
+    - schemaBaseName()
+    - "-"
+    - createdDate()
+    - "-"
+    - uuid()
+    - "-secondary"
+
```
Comment (RickHawesUSDS) [inline prime-router/metadata/file_name_templates/file-name-templates.yml:20]: Already formatted with createDate as the second field to improve sorting.

File: prime-router/settings/organizations.yml [hunk 2]
```diff
@@ -122,6 +132,24 @@
         port: 22
         filePath: ./upload
         credentialName: DEFAULT-SFTP
+    - name: elr-secondary
+      organizationName: az-phd
+      topic: covid-19
+      jurisdictionalFilter:
+        - matches(ordering_facility_state,AZ)
+      reverseTheQualityFilter: true
```
Comment (RickHawesUSDS) [inline prime-router/settings/organizations.yml:140]: Makes this a secondary feed.

File: prime-router/settings/organizations.yml [hunk 3]
```diff
@@ -122,6 +132,24 @@
...
+      translation:
+        type: CUSTOM
+        schemaName: az/az-covid-19
+        format: CSV
+        nameFormat: standard_secondary
```
Comment (RickHawesUSDS) [inline prime-router/settings/organizations.yml:145]: New file name format.

**Review by MauriceReeves-usds — APPROVED** [review]

**Pull request merged**